config/test.exs

```diff
@@ -11,6 +11,11 @@ config :cadet, environment: :test
 # Print only warnings and errors during test
 config :logger, level: :error, compile_time_purge_level: :error
 
+config :ex_aws,
+  access_key_id: "hello",
```

Comment:
What you quoted allows you to inject the id/key via `AWS_ACCESS_KEY_ID="" AWS_SECRET_ACCESS_KEY="" mix cadet.server` or fetch from ec2 instance role (that's what `:instance_role` means. I'm just hardcoding some rubbish values so that tests run. I'm using ExVCR cassettes anyway so this doesn't matter. Ah shit I should add that to `config.exs`.

Reply:
@ningyuansg Should I remove the `{:awscli, "default", 30}` item?

Reply:
Don't think it matters, since this is the year environment.

Reply:
In that case, it should be fine as long as there is an `:instance_role` entry to fail back on.